.github/workflows/php.yml

```diff
@@ -0,0 +1,49 @@
+name: Poser
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: pugx/poser:php80
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Validate composer.json and composer.lock
```

Comment:
here we don't have composer.lock versioned

Reply:
yes I can remove composer.lock